app/core/scheduler/test/test_clear_observation_info.py

```diff
@@ -24,8 +24,8 @@ def test_clear_observations():
     programs = [program_provider.parse_program(data['PROGRAM_BASIC']) for data in program_data]
 
     for program in programs:
-        for obs in program.observations():
-            Collector.clear_observation_info(obs)
+        ValidationMode._clear_observation_info(program.observations(),  
```

Comment:
Confession time: This could be a more robust test if we test the observation clearing inside the mode, creating a test with more coverage. I wanted to see if you agreed with the current structure and we can move forward with changing the test. Now is just testing your function.